bears/js/ESLintBear.py

```diff
@@ -8,8 +8,7 @@
 
 
 @linter(executable='eslint',
-        use_stdin=True,
-        use_stderr=True)
```

Comment:
Is it possible to keep `use_stderr=True`?

If that was needed for ESLint 2, it would be helpful to keep it so that anyone with ESLint 2 installed will still see any errors that were emitted by their eslint.

You could include a comment that stderr is only needed for backwards compatibility with ESLint 2, even though it is no longer supported.

Or, does `use_stderr=True` mean that some junk is captured that is difficult to exclude?